Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt

```diff
@@ -17,7 +18,9 @@ fun ViewPager2.registerOnPageChangeCallback(cb: (position: Int) -> Unit) {
 }
 
 // According to tech spec german locale only
-fun LocalDate.toFormattedDay(): String = toString("EEEE, dd.MM.yy", Locale.GERMAN)
+fun LocalDate.toFormattedDay(): String = toString("EEEE, dd.MM.yy", ConfigurationCompat.getLocales(Resources.getSystem().configuration)[0])
```

Comment:
Why are you using the systems `Configuration` object here and not our apps?

Reply:
Reverted this change back to Locale.GERMAN since Localization of date format is already addressed in #2021 (https://github.com/corona-warn-app/cwa-app-android/pull/2021). To match accessibility, I will include the changes from #2021 once the PR is merged so that the dates are read in the correct language and format when using TalkBack.